packages/kbn-reporting/common/constants.ts

```diff
@@ -16,6 +16,16 @@ export const ALLOWED_JOB_CONTENT_TYPES = [
   'text/plain',
 ];
 
+// Storage
+// Used for querying across all data stored by Reporting.
+// The wildcard avoids a 404 error in case no stored data exists.
```

Comment:
It is expected that there is no wildcard?

Reply:
Oops, the comment about the wildcard doesn't pertain to this, which should just be the name of the data stream.

The comment should be:
```
// The name of the Reporting data stream is given to query across all data stored by Reporting.
```

And maybe add:
```
// The data stream is an alias, automatically configured in the Kibana plugin of Elasticsearch, pointing to an unknown number of backing indices.
```